tsdb/wal/wal.go

```diff
@@ -46,7 +46,12 @@ const (
 // The table gets initialized with sync.Once but may still cause a race
 // with any other use of the crc32 package anywhere. Thus we initialize it
 // before.
-var castagnoliTable = crc32.MakeTable(crc32.Castagnoli)
+var (
+	castagnoliTable = crc32.MakeTable(crc32.Castagnoli)
+
+	MaxSegmentNumber = 99999999
```

Comment:
Is `math.MaxInt32`/`math.MaxInt64` more appropriate here?

Reply:
yep, but these aren't being used anymore so I will remove them